<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="PSL-DW77XM3" failures="1" tests="3" name="RUN ON InternetExplorer" time="48.138" errors="0" timestamp="2023-08-09T18:24:18 IST">
  <testcase classname="lambdaTest.TestScenario1" name="formDemo" time="6.848"/>
  <testcase classname="lambdaTest.TestScenario2" name="dragDropBySlider" time="7.497">
    <failure type="java.lang.AssertionError" message="expected [95] but found [51]">
      <![CDATA[java.lang.AssertionError: expected [95] but found [51]
at lambdaTest.TestScenario2.dragDropBySlider(TestScenario2.java:78)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- dragDropBySlider -->
  <testcase classname="lambdaTest.TestScenario3" name="inputFormSubmit" time="25.647"/>
</testsuite> <!-- RUN ON InternetExplorer -->
